torch/csrc/jit/passes/common_subexpression_elimination.cpp

```diff
@@ -9,6 +9,16 @@
 namespace torch {
 namespace jit {
 namespace {
+
+bool mayAliasGraphOutput(Node* n, const AliasDb& aliasDb) {
+  auto g_outputs = n->owningGraph()->outputs();
```

Comment:
don't define this twice?

Reply:
It's a relatively small function - where do you want me to put this?